src/main/java/com/codeborne/selenide/drivercommands/CloseDriverCommand.java

```diff
@@ -36,14 +34,14 @@ public void run() {
         t.join();
       } catch (InterruptedException e) {
         long duration = System.currentTimeMillis() - start;
-        log.log(FINE, "Failed to close webdriver " + threadId + " in " + duration + " ms", e);
+        log.debug("Failed to close webdriver " + threadId + " in " + duration + " ms", e);
```

Comment:
If you are going to use message with pattern and `{}` placeholder and parameters after comma, let's use it for each logging